src/nft/components/cart/Cart.tsx

```diff
@@ -0,0 +1,130 @@
+      top={{ sm: '0', md: 'unset' }}
+      display={bagExpanded && isNFTPage ? 'flex' : 'none'}
+    >
+      <Suspense fallback={<Loader />}>
+        <Column
+          marginTop={{ sm: '0', md: '4' }}
+          marginRight={{ sm: '0', md: '20' }}
+          className={styles.cartContainer}
+          width={{ sm: 'full', md: '288' }}
+          height={{ sm: 'full', md: 'auto' }}
+          backgroundColor="white"
+          marginLeft="0"
+          justifyContent="flex-start"
+        >
+          {isSell && sellPageState === SellPageStateType.LISTING ? null : (
+            <>
+              <BagHeader isSell={isSell} bagQuantity={isSell ? sellAssets.length : 0} />
+              {isSell ? (
+                <Column
+                  overflowX="hidden"
+                  overflowY="auto"
+                  position="relative"
+                  paddingTop="6"
+                  paddingBottom="6"
+                  height="full"
+                  className={styles.cartAssets}
+                >
+                  {sellAssets.length
+                    ? sellAssets.map((asset, index) => <CartSellAssetRow asset={asset} key={index} />)
+                    : null}
+                </Column>
+              ) : null}
+              {isSell ? (
+                <Box padding="12">
+                  <Box
+                    as="button"
+                    className={`${buttonMedium} ${orderButton}`}
+                    disabled={sellAssets.length === 0}
+                    onClick={() => {
+                      isMobile && toggleCart()
+                      setSellPageState(SellPageStateType.LISTING)
+                    }}
+                  >
+                    Continue
+                  </Box>
+                </Box>
+              ) : null}
+            </>
+          )}
+        </Column>
+      </Suspense>
+    </Box>
+  )
+}
+
+const BagHeader = ({ bagQuantity, isSell }: { bagQuantity: number; isSell: boolean }) => {
+  const toggleCart = useBag((state) => state.toggleBag)
+  const resetSellAssets = useSellAsset((state) => state.reset)
+  const isMobile = useIsMobile()
+  return (
+    <Box position="relative" zIndex="2" paddingTop="20" paddingLeft="12" paddingRight="12">
+      {isMobile ? (
+        <Box
+          as="button"
+          border="none"
+          color="darkGray"
+          background="black"
+          className={styles.closeIcon}
+          onClick={toggleCart}
+        >
+          <CloseDropDownIcon />
+        </Box>
+      ) : null}
+      <Box className={headlineSmall} paddingTop="0" paddingBottom="8" paddingX="0" margin="0">
+        {isSell ? 'Selected items' : 'My Bag'}
+      </Box>
+      {bagQuantity > 0 ? (
+        <Box className={bodySmall} paddingTop="0" paddingBottom="8" paddingX="0" marginY="0" marginX="auto">
+          {bagQuantity} {bagQuantity === 1 ? 'NFT' : 'NFTs'}
+          <span style={{ fontSize: '8px', position: 'relative', top: '-2px', padding: '0 2px 0 4px' }}>&#x2022;</span>
```

Comment:
this can be changed to `<Box as="span" position="relative" paddingTop="2" paddingLeft="2" paddingRight="4" style={{ fontSize: '8px' }}>•</Box>`

Reply:
We still need the -2 position and your left and right are swapped so it became: `<Box as="span" position="relative" paddingRight="2" paddingLeft="4" style={{ fontSize: '8px', top: '-2px' }}>`

Reply:
Ah yes you're right - css is dumb I feel like left -> right is way more intuitive than right -> left

Reply:
I always remember it as clockwise staring with 0/12 position ie top then going right (3), bottom (6), left (9)